ncvoter/urls.py

```diff
@@ -1,7 +1,8 @@
 """ncvoter URL Configuration"""
-from django.conf.urls import url
+from django.urls import path, include
 from django.contrib import admin
 
 urlpatterns = [
-    url(r'^admin/', admin.site.urls),
+    path(r'admin/', admin.site.urls),
+    path(r'api/v1/', include('voter.urls'))
```

Comment:
Since these aren't regexes anymore, they can be regular strings instead of raw string literals.